src/worker/lambda/zygote/importCache.go

```diff
@@ -233,15 +249,20 @@ func (cache *ImportCache) getSandboxInNode(node *ImportCacheNode, forceNew bool,
 			}
 		}
 		node.sbRefCount += 1
+		fmt.Printf("node.sb != nil: node %d, getSandboxInNode with ref count %d\n", node.SplitGeneration, node.sbRefCount)
```

Comment:
We ought to be making log.Printf calls so we get timestamps too.  fmt.Printf would only be for commands a user is executing (not the long-running server).